README.md

```diff
@@ -59,6 +59,21 @@ func main() {
 }
 ```
 
+### Custom aggregation for reported errors
+
+By default errors are aggregated by their _stack trace_ and _error message_. This might cause that errors that apparently are the same are treated as different in Periskop.
```

Comment:
could we give an example of this?

Reply:
How? with a screenshot or what?

Reply:
Yeah; i think a screenshot would suffice